packages/web/src/components/ui/PackageVersion/PackageVersion.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { useState } from 'react';
+import styles from './PackageVersion.module.scss';
+import clsx from 'clsx';
+import { Icon } from '../Icon/Icon';
+import CircularProgress from '../CircularProgress/CircularProgress';
+import ProgressBar from '../ProgressBar/ProgressBar';
+import { formatNumber } from 'utils/helpers';
+
+type Module = {
+  path: string;
+  moduleSize: number;
+  moduleSizeUnit: string;
+  percentage: number;
+};
+
+type EntryPoint = {
+  value: number;
+  code: string;
+  entry: string;
+};
+
+type Props = {
+  version: string;
+  updateDate: string;
+  uses: number;
+  size: number;
+  sizeUnitShorthand: string;
+  sizeUnit: string;
```

Comment:
Also regarding these, I'd always resort to passing everything around in bytes and formatting in-place after. We should already have a function to format bytes to readable size laying around.